addon/components/code-inline.js

```diff
@@ -1,3 +1,5 @@
+/* global Prism */
```

Comment:
Indeed, it is there to tell `jshint` not to throw errors about a globally defined variable. Before this fix, when you try `ember serve` you should see the errors thrown in the console, or when visiting `localhost:4200/tests` the tests should fail because of `jshint` errors.